dev-libs/libmcrypt/libmcrypt-2.5.8.recipe

```diff
@@ -0,0 +1,78 @@
+SUMMARY="Replacements for the old Unix crypt"
+DESCRIPTION="libmcrypt are intended to be replacements for the \
+old Unix crypt, except that they are under the GPL and \
+support an ever-wider range of algorithms and modes."
```

Comment:
Hmm, "libmcrypt are intended" does not sound very well. Shall we use the following description, instead?
```
DESCRIPTION="libmcrypt is the companion library used by Mcrypt, which is \
itself an enhanced replacement for the old crypt command.
libmcrypt allows developers to use a wide range of encryption functions \
without making drastic changes to their code. It contains the actual \
encryption functions and provides a standardized mechanism for accessing \
them."
```
It's based on the description found in http://mcrypt.sourceforge.net/